src/structlog/processors.py

```diff
@@ -531,3 +543,184 @@ def __call__(
             )
 
         return event_dict
+
+
+class CallsiteParameter(str, enum.Enum):
+    """
+    Callsite parameters that can be added to an event dictionary with the
+    `structlog.processors.CallsiteParameterAdder` processor class.
+
+    The string values of the members of this enum will be used as the keys for
+    the callsite parameters in the event dictionary.
+
+    .. versionadded:: 21.5.0
+    """
+
+    #: The full path to the python source file of the callsite.
```

Comment:
why the `:`s in each line?

Reply:
This is so the descriptions show up in sphinx.

Without the `:` the sphinx output looks like this:

![image](https://user-images.githubusercontent.com/4481387/146336424-e4a862ad-5618-4333-9f08-b337f7cf2a0e.png)

With the `:` the sphinx output looks like this:

![image](https://user-images.githubusercontent.com/4481387/146336469-74e98f1f-f8de-45c4-ba36-f50d606fc1a4.png)

Reply:
OMG TIL!